Marlin/SanityCheck.h

```diff
@@ -352,11 +352,22 @@
     #error "EXTRUDERS must be 1 with HEATERS_PARALLEL."
   #endif
 
+#elif ENABLED(MK2_MULTIPLEXER)
+  #error "MK2_MULTIPLEXER requires 2 or more EXTRUDERS."
 #elif ENABLED(SINGLENOZZLE)
   #error "SINGLENOZZLE requires 2 or more EXTRUDERS."
 #endif
 
 /**
+ * Sanity checking for the Průša MK2 Multiplexer
+ */
+#ifdef SNMM
+  #error "SNMM is now MK2_MULTIPLEXER. Please update your configuration."
+#elif ENABLED(MK2_MULTIPLEXER) && DISABLED(ADVANCED_PAUSE_FEATURE)
```

Comment:
I don't see why `ADVANCED_PAUSE_FEATURE` is required for `MK2_MULTIPLEXER`. The multiplexer allows for expansion of extruder steppers, but they don't necessarily have to be in a Bowden configuration as used in the MK2. There's no reason why someone couldn't use the multiplexer on a direct-feed printer to allow for multiple extruders to be driven from a single driver.